src/pat/autotoc/autotoc.js

```diff
@@ -107,10 +137,28 @@ export default Base.extend({
                     $(this).trigger("clicked");
                     if (!options.skipHash) {
                         if (window.history && window.history.pushState) {
-                            window.history.pushState({}, "", "#" + $el.attr("id"));
+                            window.history.pushState({}, "", `#${id}`);
                         }
                     }
                 });
+            if (asTabs) {
+                $nav.addClass([
+                    "mt-0",
+                    "me-2",
+                    "ms-2",
+                    "py-2",
+                    "px-3",
+                    "border-bottom-0",
+                ]);
+            }else{
+                $nav.addClass([
+                    "text-decoration-underline",
+                    "p-0",
+                    "my-0",
+                    "me-3",
+                    `ms-${self.getLevel($level)+2}`
```

Comment:
As you've mentioned correctly, our indentation is custom CSS, so I would add the `autotoc-level-x ` classes here explicitly and add keep css definitions in the `autotoc.scss` file. The base margin class would be `mx-3` (`my` is not relevant)